src/richie-front/js/components/SearchFilter/CloseIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
```

Comment:
Can we prepare the project for other icons to come by moving this component to something like `src/richie-front/js/components/icons/CloseIcon.tsx`?

Reply:
This is a good idea. It will make it simpler to change things when we decide to standardize icons